website/pages/docs/overview/roadmap.mdx

```diff
@@ -0,0 +1,14 @@
+# The Future of Mix
+We are excited to announce the release of Mix 1.0.0! This release is a major milestone in the development of Mix, and we are thrilled to share it with you. The version 1.0.0 signifies that we have reached a stable API, and we are confident in this API. Through this Roadmap, we will share our plans for the future of Mix.
+
+## Use cases for Mix
+We've been working on Mix for a long time to solve the styling problem in Flutter. Part of building a solution is showing you how Mix was planned to be used. Because of this, we will spend some time creating real projects using Mix.
+
+## Our own Design System: Remix
+We are also planning to create a Design System based on Mix. This Design System will be called **Remix**. The goal of this Design System is to be completely styleable and flexible in the same way as Mix. You can follow the progress of this project on [GitHub](https://github.com/conceptadev/remix_ui).
+
+## A custom lint for Mix
+While Mix was being created, we took note of the best practices and the common mistakes that developers make when using Mix. That's why we want to create a custom lint for Mix. This lint will help developers avoid common mistakes and use Mix in a better way.
+
+## Validation of codegen approach
+Mix, by default, requires a lot of boilerplate code to work, especially when you need to create a custom `StyleWidget`. We are planning to use a codegen approach to generate this boilerplate code.
```

Comment:
Why not design for the future and use the macro system that will be released in 6 months? https://dart.dev/language/macros

`build_runner` improvements have largely been abandoned for this reason.